packages/app/src/store/webviewRef/courseWebviewStore/index.ts

```
@@ -0,0 +1,14 @@
+import { create } from "zustand";
+
+interface WebviewRef {
+  canGoBack: boolean;
+  setCanGoBack?: (canGoBack: boolean) => void;
+}
+
+/**
+ * 해당 스토어는 코스 웹뷰에서 뒤로가기 가능 여부를 관리하는 용도입니다.
+ */
+export const useCourseWebviewStore = create<WebviewRef>((set, get) => ({
```

ESLint

[error] 'get' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)